<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cddangxuat_form</class>
 <widget class="QMainWindow" name="cddangxuat_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Đăng xuất</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(224, 243, 253);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_return">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(154, 205, 244);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/ic_return.png</normaloff>images/ic_return.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(154, 205, 244);
font: 16pt &quot;Cambria&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Tài khoản và bảo mật&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_doimk">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>261</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
font: 13pt &quot;Cambria&quot;;
background-color: rgb(146, 214, 255);</string>
    </property>
    <property name="text">
     <string>Đăng xuất</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
</string>
    </property>
    <property name="text">
     <string>Bạn xác nhận đăng xuất khỏi thiết bị?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_huybo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 194, 209);</string>
    </property>
    <property name="text">
     <string>Hủy bỏ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_dangxuat">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 194, 209);</string>
    </property>
    <property name="text">
     <string>Đăng xuất</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>btn_return</zorder>
   <zorder>lbl_doimk</zorder>
   <zorder>label_2</zorder>
   <zorder>btn_huybo</zorder>
   <zorder>btn_dangxuat</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
